var/spack/repos/builtin/packages/geopm-runtime/package.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class GeopmRuntime(AutotoolsPackage):
+    """The Global Extensible Open Power Manager (GEOPM) Runtime is designed to
+    enhance energy efficiency of applications through active hardware
+    configuration."""
+
+    homepage = "https://geopm.github.io"
+    git = "https://github.com/geopm/geopm.git"
+    url = "https://github.com/geopm/geopm/tarball/v3.0.1"
+
+    maintainers("bgeltz", "cmcantalupo")
+    license("BSD-3-Clause")
+    tags = ["e4s"]
+
+    version("develop", branch="dev", get_full_repo=True)
+    version("3.0.1", sha256="32ba1948de58815ee055470dcdea64593d1113a6cad70ce00ab0286c127f8234")
+
+    variant("debug", default=False, description="Enable debug")
+    variant("docs", default=True, description="Create man pages with Sphinx")
+    variant("overhead", default=False, description="Track time spent in GEOPM API calls")
+    variant("beta", default=False, description="Enable beta features")
+    variant("mpi", default=True, description="Enable MPI dependent components")
+    variant("fortran", default=True, description="Build fortran interface")
+    variant("openmp", default=True, description="Build with OpenMP")
+    variant("ompt", default=True, description="Use OpenMP Tools Interface.")
+    variant("gnu-ld", default=False, description="Assume C compiler uses gnu-ld.")
+    variant("intel", default=True, description="Build with the Intel compilers")
```

Comment:
We don't use variants to control compilers, we use `%oneapi` instead. I would remove this variant.

Reply:
This variant was really about including support for Intel MKL in some of GEOPM's tools.  I'll reword.